capstone/src/main/webapp/youtubeScript.js

```diff
@@ -264,9 +286,66 @@ function showCategories() {
     radiobox.onclick = function() {
       enableTextInput(this);   
     }
-    const container = document.getElementById('category-container');
-    container.appendChild(radiobox);
-    container.appendChild(label);
-    container.appendChild(document.createTextNode (" "));
+    const categoryContainer = document.getElementById('category-container');
+    categoryContainer.appendChild(radiobox);
+    categoryContainer.appendChild(label);
+    categoryContainer.appendChild(document.createTextNode(" "));
   }
 }
+
+/** Converts perspective results to knoop scale then to mohs*/
+function perspectiveToxicityScale(attributeAverages) {
```

Comment:
nit: Let's use a more verb-y name for this (e.g. `scaleAverages()` or `convertToMohsScale()` or something like that).

Reply:
addressed in "optimize and clarify PerspectiveToxicityScale function"